template.go

```diff
@@ -0,0 +1,26 @@
+package cloudconfig
+
+import (
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/k8scloudconfig/v_0_1_0_3500632"
+)
+
+type Template struct {
+	Master string
+	Worker string
+}
+
+func NewTemplate(v string) (Template, error) {
```

Comment:
This would be the interface for library users to get versioned templates.

Reply:
I'd like to see `type Version string` with constants. It would introduce some level of type safety.

Reply:
Good idea. Done.